internal/webhook/v1beta2/integrationtestscenario_webhook.go

```diff
@@ -96,3 +99,29 @@ func (v *IntegrationTestScenarioCustomValidator) ValidateDelete(ctx context.Cont
 
 	return nil, nil
 }
+
```

Comment:
Those changes feels like duplication of code above, is it needed?

Reply:
No, I'll remove them. Thanks for the reminder